frontend/pages/hosts/details/cards/Software/SoftwareTableConfig.tsx

```diff
@@ -207,34 +213,55 @@ const generateSoftwareTableHeaders = (deviceUser = false): IDataColumn[] => {
         );
       },
     },
-    // TODO: Enable after backend has been updated to provide last_opened_at
-    // {
-    //   title: "Last used",
-    //   Header: (cellProps) => (
-    //     <HeaderCell
-    //       value={cellProps.column.title}
-    //       isSortedDesc={cellProps.column.isSortedDesc}
-    //     />
-    //   ),
-    //   accessor: "last_opened_at",
-    //   Cell: (cellProps) => {
-    //     const lastUsed = isNaN(Date.parse(cellProps.cell.value))
-    //       ? "Unavailable"
-    //       : `${distanceInWordsToNow(Date.parse(cellProps.cell.value))} ago`;
-    //     return (
-    //       <span
-    //         className={
-    //           lastUsed === "Unavailable"
-    //             ? "software-last-used-muted"
-    //             : "software-last-used"
-    //         }
-    //       >
-    //         {lastUsed}
-    //       </span>
-    //     );
-    //   },
-    //   sortType: "dateStrings",
-    // },
+    {
+      title: "Last used",
+      Header: (cellProps) => (
+        <HeaderCell
+          value={cellProps.column.title}
+          isSortedDesc={cellProps.column.isSortedDesc}
+        />
+      ),
+      accessor: "last_opened_at",
+      Cell: (cellProps: ILastUsedCellProps): JSX.Element => {
+        const lastUsed = cellProps.cell.value
+          ? `${formatDistanceToNow(Date.parse(cellProps.cell.value))} ago`
+          : "Unavailable";
+        const hasLastUsed = lastUsed !== "Unavailable";
+        return (
+          <>
+            <span
+              className={`last-used ${
+                lastUsed === "Unavailable" ? "text-muted" : ""
+              }`}
+              data-tip
+              data-for={`last_used__${cellProps.row.index}`}
+              data-tip-disable={hasLastUsed}
+            >
+              {lastUsed}
+            </span>
+            {!hasLastUsed && (
```

Comment:
I believe our pattern up to now had been to include the tooltip element without a conditional since the `data-tip-disable` property controls whether it is visible. But I'm not aware of any good reason why that pattern took hold. Were there any specific reasons for using the conditional here (other than good code hygiene)? I wonder if there are tradeoffs in terms of how the virtual DOM renders the tooltip. More just curiosity on my part.

Reply:
I didn't realize there was a `data-tip-disable` property, thanks for the heads up! Updated.